decidim-proposals/db/migrate/20210127115628_fix_answered_proposals_after_copy.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class FixAnsweredProposalsAfterCopy < ActiveRecord::Migration[5.2]
+  def change
+    proposals_after_copy = Decidim::ResourceLink.where(from_type: "Decidim::Proposals::Proposal").pluck(:from_id)
+
+    result = Decidim::Proposals::Proposal.where.not(state_published_at: nil).where(state: nil, id: proposals_after_copy)
```

Comment:
Can we make a single update?

```
result.update_all(state_published_at: nil)
```

Reply:
`update_all` doesn't trigger callbacks. I wanted to make sure they were triggered to run the `SearchableResource` methods to update the search indexs.